### ISSUE TITLE
TensorBoard not displaying any data for the model

### ISSUE THREAD

**Issue by w0330t** [freqtrade/freqtrade#9615]
## Describe your environment

  * Operating system: WSL2 ArchLinux
  * Python Version: Python 3.10.13
  * CCXT version: 4.1.98
  * Freqtrade Version: 2024.1-dev-3e7d5bbae
  
## Your question

I'm currently using the freqtrade project and I've run into an issue with TensorBoard. I've followed the official documentation to set up my simulation trading:

```shell
freqtrade trade --config config_examples/config_freqai.example.json --strategy FreqaiExampleStrategy --freqaimodel LightGBMRegressor --strategy-path freqtrade/templates
```

After running the simulation, I attempted to use TensorBoard to visualize the model's performance using the following command:
```shell
tensorboard --logdir user_data/models/uniqe-id
```
However, when I access http://localhost:6006/, I receive the following message:

```
No dashboards are active for the current data set.
Probable causes:

You haven’t written any data to your event files.
TensorBoard can’t find your event files.
If you’re new to using TensorBoard, and want to find out how to add data and set up your event files, check out the README and perhaps the TensorBoard tutorial.
If you think TensorBoard is configured properly, please see the section of the README devoted to missing data problems and consider filing an issue on GitHub.

Last reload: Dec 31, 2023, 2:18:16 PM
Log directory: user_data/models/uniqe-id
```

I've double-checked the file path and it appears to be correct. My `user_data` directory structure is as follows:

```shell
tree
.
|-- backtest_results
|-- config
|-- data
|   `-- binance
|       `-- futures
|           |-- 1INCH_USDT_USDT-15m-futures.feather
|           |-- 1INCH_USDT_USDT-1h-futures.feather
|           |-- 1INCH_USDT_USDT-3m-futures.feather
|           |-- 1INCH_USDT_USDT-8h-funding_rate.feather
|           |-- 1INCH_USDT_USDT-8h-mark.feather
|           |-- ALGO_USDT_USDT-15m-futures.feather
|           |-- ALGO_USDT_USDT-1h-futures.feather
|           |-- ALGO_USDT_USDT-3m-futures.feather
|           |-- ALGO_USDT_USDT-8h-funding_rate.feather
|           |-- ALGO_USDT_USDT-8h-mark.feather
|           |-- BTC_USDT_USDT-15m-futures.feather
|           |-- BTC_USDT_USDT-1h-futures.feather
|           |-- BTC_USDT_USDT-3m-futures.feather
|           |-- BTC_USDT_USDT-8h-funding_rate.feather
|           |-- BTC_USDT_USDT-8h-mark.feather
|           |-- ETH_USDT_USDT-15m-futures.feather
|           |-- ETH_USDT_USDT-1h-futures.feather
|           |-- ETH_USDT_USDT-3m-futures.feather
|           |-- ETH_USDT_USDT-8h-funding_rate.feather
|           `-- ETH_USDT_USDT-8h-mark.feather
|-- freqaimodels
|-- hyperopt_results
|-- hyperopts
|   `-- sample_hyperopt_loss.py
|-- logs
|-- models
|   `-- uniqe-id
|       |-- global_metadata.json
|       |-- historic_predictions.backup.pkl
|       |-- historic_predictions.pkl
|       |-- pair_dictionary.json
|       |-- run_params.json
|       |-- sub-train-1INCH_1704003921
|       |   |-- cb_1inch_1704003921_metadata.json
|       |   |-- cb_1inch_1704003921_model.joblib
|       |   |-- cb_1inch_1704003921_trained_dates_df.pkl
|       |   `-- cb_1inch_1704003921_trained_df.pkl
|       |-- sub-train-1INCH_1704003928
|       |   |-- cb_1inch_1704003928_feature_pipeline.pkl
|       |   |-- cb_1inch_1704003928_label_pipeline.pkl
|       |   |-- cb_1inch_1704003928_metadata.json
|       |   |-- cb_1inch_1704003928_model.joblib
|       |   |-- cb_1inch_1704003928_trained_dates_df.pkl
|       |   `-- cb_1inch_1704003928_trained_df.pkl
|       |-- sub-train-1INCH_1704003935
|       |   |-- cb_1inch_1704003935_feature_pipeline.pkl
|       |   |-- cb_1inch_1704003935_label_pipeline.pkl
|       |   |-- cb_1inch_1704003935_metadata.json
|       |   |-- cb_1inch_1704003935_model.joblib
|       |   |-- cb_1inch_1704003935_trained_dates_df.pkl
|       |   `-- cb_1inch_1704003935_trained_df.pkl
|       |-- sub-train-ALGO_1704003925
|       |   |-- cb_algo_1704003925_feature_pipeline.pkl
|       |   |-- cb_algo_1704003925_label_pipeline.pkl
|       |   |-- cb_algo_1704003925_metadata.json
|       |   |-- cb_algo_1704003925_model.joblib
|       |   |-- cb_algo_1704003925_trained_dates_df.pkl
|       |   `-- cb_algo_1704003925_trained_df.pkl
|       `-- sub-train-ALGO_1704003932
|           |-- cb_algo_1704003932_feature_pipeline.pkl
|           |-- cb_algo_1704003932_label_pipeline.pkl
|           |-- cb_algo_1704003932_metadata.json
|           |-- cb_algo_1704003932_model.joblib
|           |-- cb_algo_1704003932_trained_dates_df.pkl
|           `-- cb_algo_1704003932_trained_df.pkl
|-- notebooks
|-- plot
|-- strategies
|   `-- sample_strategy.py
`-- tradesv3.dryrun.sqlite-shm
```

From the directory structure, it seems like the model files are present in the correct directory. However, TensorBoard is still unable to find or read the event files.

Could you please help me understand why this might be happening and how I can resolve it? Any help would be greatly appreciated.

Thank you.

**Comment by robcaulk** [comment]
Hello,

I do not believe LightGBM has easy tensorboard support.

On the other hand, Catboost and XGBoost are both drop in replacements and *do* have tensorboard support, and are available for metric tracking in FreqAI.

Cheers,

Robert
